nimble/host/src/ble_hs_hci_evt.c

```diff
@@ -341,6 +347,8 @@ ble_hs_hci_evt_le_conn_complete(uint8_t subevent, uint8_t *data, int len)
                         BLE_HS_LOG(DEBUG, "Memory unavailable for new peer record\n");
                     }
                 }
+                /* Set the correct ota address */
```

Comment:
This can be avoided !!

Reply:
I'm unsure if you want the comment or memcpy() or both removed here, it's a nice to have for logging, but not a big deal. Please clarify 😃.

Reply:
Nevermind, I will remove as the code to be removed in `ble_hs_conn.c` would be required to support it.

Reply:
I meant both as it will be taken care after removing https://github.com/espressif/esp-nimble/pull/7/files#diff-1fa24060fedfaf880b562444ad37488cR328